.github/template_specific/README_template.md

```diff
@@ -11,36 +11,57 @@ Short description of this module.
 ## Content
 1. [Compatibility](#compatibility)
 2. [Requirements](#requirements)
-    1. [Required software](#required-software)
-    2. [Other](#other)
-2. [Usage](#usage)
-    1. [Providers](#providers)
-3. [Inputs](#inputs)
-4. [Outputs](#outputs)
-5. [Example](#example)
+    1. [Required modules](#required-modules)
+    2. [Required software](#required-software)
+    3. [Other](#other)
+3. [Usage](#usage)
     1. [Verifying setup](#verifying-setup)
-6. [Contributors](#contributors)
-7. [License](#license)
+    2. [Providers](#providers)
+4. [Example usage](#example-usage)
+5. [Inputs](#inputs)
+6. [Outputs](#outputs)
+7. [Contributors](#contributors)
+8. [License](#license)
 
 ## Compatibility
-List of compatible versions. E.g.
-1. Terraform 0.12.2 or newer
-2. Nomad 0.12 or newer
+| Software | OSS Version | Enterprise Version |
+|:---|:---|:---|
+| Terraform | 0.13.1 or newer ||
+| Nomad | 0.12.3 or newer ||
+
+Any other?
 
 ## Requirements
+
+### Required modules
+| Module | Version |
+| :----- | :------ |
+| None | - |
+
 ### Required software
 1. Software 1
 2. Software 2
 
 ### Other
-Any other requirements. E.g. "This needs to be run on a Debian system"
+Any other requirements? E.g. "This needs to be run on a Debian system"
```

Comment:
Why `debian`? We successfully ran on MacOs and Ubuntu

Reply:
This was already implemented, but I might be better if we just remove the `E.g. ...` part of it?